### ISSUE TITLE
Right-click doesn't clear selection created with tableselection plugin

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor-dev#1264]
## Are you reporting a feature request or a bug?

Bug

## [Check if the issue is already reported](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_issues_readme-section-avoid-duplicates)

*Put all reference links here…*

## Provide detailed reproduction steps (if any)

1. Insert below HTML into CKEditor
```
<table border="1" cellpadding="1" cellspacing="1" style="width:500px">
	<tbody>
		<tr>
			<td>11</td>
			<td>12</td>
			<td>13</td>
		</tr>
		<tr>
			<td>21</td>
			<td>22</td>
			<td>23</td>
		</tr>
	</tbody>
</table>
```
2. Select Cell ```22``` so that whole cell is selected (blue) 
3. Right-click inside cell ```12``` and select Column -> Insert Column After.

### Expected result

New column gets inserted between first and second column.

### Actual result

New column gets inserted between second and third column.
It is also worth mentioning that when left-clicking in cell ```22``` and then right-clicking in cell ```12```, selection is moved properly and new cell is inserted as expected.

## Other details

* Browser: Any
* OS: Any
* CKEditor version: 4.7.0
* Installed CKEditor plugins: tableselection

**Comment by jswiderski** [comment]
Sample `mp4` showing the problem: [tableselection.zip](https://github.com/ckeditor/ckeditor-dev/files/1513277/tableselection.zip)

**Comment by Comandeer** [comment]
Merged into 4.10.2.
